### ISSUE TITLE
Adam in tf1.x

### ISSUE THREAD

**Issue by YitaoQiu** [huijiancai99/PINN-Euler-Bernoulli#2]
self.global_step = tf.Variable(0, trainable=False)
        self.learning_rate = tf.train.exponential_decay(_INITIAL_LEARNING_RATE,
                                                        self.global_step,
                                                        _DECAY_STEPS, 
                                                        _DECAY_RATE, 
                                                        staircase=False)
        self.optimizer_Adam = tf.train.AdamOptimizer(self.learning_rate, beta1=0.99)

        self.train_op_Adam = self.optimizer_Adam.minimize(self.loss, global_step=self.global_step)
